### PR TITLE
add copyrights and licenses

### PR DESCRIPTION
**Pull request by kaddaSz**
- copyright headers added to files
- reuse tool configured
- THIRD-PARTY-NOTICES added

### CODE REVIEW

File: Sources/jsonfunctions/Expressions/_Expression.swift [hunk 1]
```diff
@@ -1,6 +1,8 @@
 //
 // json-functions-swift
 //
+// Copyright (c) 2022 SAP SE or an SAP affiliate company
```
Comment (ArturFriesen) [inline Sources/jsonfunctions/Expressions/_Expression.swift]: Expression was moved from Parser and modified.

File: Sources/jsonfunctions/JSON/Date.swift [hunk 2]
```diff
@@ -1,7 +1,5 @@
 // Copyright (c) 2021 Patrick Amrein <amrein@ubique.ch>
-// 
-// Licensed under LGPL
```
Comment (ArturFriesen) [inline Sources/jsonfunctions/JSON/Date.swift:3]: Is this ok?

**Review by ArturFriesen — CHANGES_REQUESTED** [review]

File: Sources/jsonfunctions/JSON/Date.swift (hunk 2 above)
Comment (kaddaSz) [reply Sources/jsonfunctions/JSON/Date.swift:3]: I checked it with open source team.. They confirmed it.
Complete original repo isn't under LGPL anymore but under MIT.

File: Sources/jsonfunctions/Expressions/_Expression.swift (hunk 1 above)
Comment (kaddaSz) [reply Sources/jsonfunctions/Expressions/_Expression.swift]: ok, will change it..

**Review by ArturFriesen — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

**Pull request merged**